argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 To use Argo Rollouts and manage progressive delivery, after you install the {gitops-titel} Operator on the cluster, you can create and configure a `RolloutManager` custom resource (CR) instance in the namespace of your choice. You can scope the `RolloutManager` CR for single or multiple namespaces. 
 
-[id="prerequisites_using-argo-rollouts-for-progressive-deployment-delivery"]
+[id="prerequisites_using-argo-rollouts-for-progressive-deployment-delivery_{context}"]
```

Comment:
Please see how to anchor [ "Prerequisites", "Additional resources", and "Next steps" titles in assemblies](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies).